2023/11/01 05:40:05 failed to dial tcp connection, retrying
2023/11/01 05:40:06 failed to dial tcp connection, retrying
[2023-11-01T05:40:06.609Z] info  code-server 4.11.0 85e083580dec27ef19827ff42d3c9257d56ea7e3
[2023-11-01T05:40:06.610Z] info  Using user-data-dir ~/.local/share/code-server
[2023-11-01T05:40:06.629Z] info  Using config file ~/.config/code-server/config.yaml
[2023-11-01T05:40:06.629Z] info  HTTP server listening on http://127.0.0.1:8080/
[2023-11-01T05:40:06.629Z] info    - Authentication is disabled
[2023-11-01T05:40:06.629Z] info    - Not serving HTTPS
====================================>token: 54718e55-d508-462b-997b-3194d65c328e
[05:40:09] 




[05:40:10] Extension host agent started.
[05:40:11] [127.0.0.1][4f4b809e][ManagementConnection] New connection established.
[05:40:12] [127.0.0.1][61362868][ExtensionHostConnection] New connection established.
[05:40:12] [127.0.0.1][61362868][ExtensionHostConnection] <377> Launched Extension Host Process.
[05:40:15] Using the in-memory credential store as the operating system's credential store could not be accessed. Please see https://aka.ms/vscode-server-keyring on how to set this up. Details: libsecret-1.so.0: cannot open shared object file: No such file or directory
[05:40:25] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /opt/conda/lib/python3.8)
[2023-11-01T05:42:22.646Z] error Failed to get latest version [38;2;140;140;140m{"error":"connect ETIMEDOUT 20.205.243.168:443"}[0m
[07:13:11] [127.0.0.1][4f4b809e][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[07:13:11] [127.0.0.1][61362868][ExtensionHostConnection] <377> Extension Host Process exited with code: 0, signal: null.
2023/11/01 07:23:12 exit program after timeout
